pkg/agent/util/ethtool.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+const (
+	IFNAMSIZ        = 16         // defined in linux/if.h
+	SIOCETHTOOL     = 0x8946     // ethtool interface, defined in linux/sockios.h
+	ETHTOOL_STXCSUM = 0x00000017 // set TX hw csum enable, defined in linux/ethtool.h
```

Comment:
Should we follow go convention for constant? https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.
though apparently the package syscall doesn't respect it.

Reply:
I think `syscall` doesn't follow the convention for "well-known" Linux constants, so I'm tempted to do the same thing.